CSP/read_problem.py

```diff
@@ -0,0 +1,71 @@
+# This file will handle the reading in of a problem in 4pw format into our CSP model
+
+from constraint import *
+from find_domain import find_domain
+from rules import generate_constraints
+import sys
+
+
+# Helper function that parses a 3 character harmony string into a len 3 list (e.g. 5Mr -> [5, 'M', 'r'])
+def parse_harmony(harmony_string):
+    return [int(harmony_string[0]) if harmony_string[0].isdigit() else harmony_string[0], harmony_string[1],
+            int(harmony_string[2]) if harmony_string[2].isdigit() else harmony_string[2]]
```

Comment:
Separate into 3 lines to make clear that 0 and 1 are not linked?  I just messed up.